app/grandchallenge/reader_studies/views.py

```diff
@@ -896,7 +896,7 @@ def ground_truth(self, request, pk=None, case_pk=None):
         )
 
 
-class QuestionViewSet(ReadOnlyModelViewSet):
+class QuestionViewSet(ModelViewSet):
```

Comment:
This seems like a brutal security issue to me. Is there a better way to allow _patching_ of the Question-model?

I only want to rewrite the "order" field of the Question-model.

Reply:
Yes, this change bypasses all security controls that means anyone can do anything to a question.